t/ui/15-admin-workers.t

```diff
@@ -124,6 +124,10 @@ subtest 'worker overview' => sub {
     # check worker 2
     is($driver->find_element('tr#worker_2 .worker')->get_text(), 'remotehost:1', 'remotehost:1 shown');
     $driver->find_element('tr#worker_2 .help_popover')->click();
+    for (0 .. 3) {
+        last if $driver->find_element('.popover')->get_text() =~ qr/Worker status\nJob: 99961/;
```

Comment:
btw: selenium does the loop itself if you give it a chance. So instead of `$driver->find_element(.'popover')->get_text() =~ ... ` you need to call `$driver->find_element_by_xpath('//div[@class="popover" and contains(text(), "Job: 99961")]')`